src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4344,9 +4344,8 @@ void CalcHeatEmissionReport(EnergyPlusData &state)
                                                          state.dataCondenserLoopTowers->towers(iCooler).FanEnergy +
                                                          state.dataCondenserLoopTowers->towers(iCooler).BasinHeaterConsumption;
     }
-    for (int iCooler = 1; iCooler <= state.dataEvapFluidCoolers->NumSimpleEvapFluidCoolers; ++iCooler) {
-        state.dataHeatBal->SysTotalHVACRejectHeatLoss += state.dataEvapFluidCoolers->SimpleEvapFluidCooler(iCooler).Qactual * TimeStepSysSec +
-                                                         state.dataEvapFluidCoolers->SimpleEvapFluidCooler(iCooler).FanEnergy;
+    for (auto &thisCooler : state.dataEvapFluidCoolers->SimpleEvapFluidCooler) {
+        state.dataHeatBal->SysTotalHVACRejectHeatLoss += thisCooler.Qactual * TimeStepSysSec + thisCooler.FanEnergy;
```

Comment:
We don't keep `NumSimpleEvapFluidCoolers` on state anymore, but we don't need to.  Besides, this range-based loop is just sleek this way, and clearly shows what it is doing, without having to worry about any indexes or lengths of arrays.